### DISCUSSION TITLE
Saving translation

### DISCUSSION THREAD

**Discussion by asp4translation** [AI4Bharat/Chitralekha#7] [Q&A]
I am working on a translation task assigned by NPTEL. Modifications to translation works fine on page 1. Once I complete page 2, save changes and move on to page 3, the contents in page 2 reverts to original, including any merges.  Is this a bug?
OS: macOS Monterey
Browser: Firefox 104.0.1

**Comment by aravinth** [comment]
@asp4translation - Can you confirm if the issue still persists?

**Reply by Parumaruthy** [reply]
This problem persists and is taking too long to work on it. Correcting the pages is difficult as the changes I make soon disappear when I change the cursor position.
